Sources/App/Commands/Analyzer.swift

```diff
@@ -238,7 +252,9 @@ func getManifests(logger: Logger,
             let errors = m.compactMap { $0.getError() }
                 .map { AppError.genericError(pkg.id, "getManifests failed: \($0.localizedDescription)") }
             errors.forEach { logger.report(error: $0) }
-            guard !successes.isEmpty else { return .failure(AppError.noValidVersions(pkg.id, pkg.url)) }
+            if !versions.isEmpty && successes.isEmpty {
+                return .failure(AppError.noValidVersions(pkg.id, pkg.url))
+            }
```

Comment:
Our tests nicely flagged that previously we blanked assumed getManifests to fail if there were no successful manifest reads. Because we can how call this for a package with no versions (i.e. there were no additions), we also need to ensure that there was something to look for in the first place and only then error out.